magprime/templates/regextra.html

```diff
@@ -77,9 +77,21 @@
         {% endif %}
 
         // MAGFest uses tiered badge pricing based on caps, so we want to link to our FAQ explaining this
-        if ($('.prereg-price-notice').size()) {
-            $('#reg-types').append("<div class='help-block col-sm-9 col-sm-offset-3'>Prices may increase before the set date. {{ macros.popup_link("http://magfest.org/faq#-KOpcFnNhdsUIqvvf6VB", "Why is this?")|e }}</div>");
-        }
+        {% if c.BADGES_LEFT_AT_CURRENT_PRICE %}
+            {% if c.BADGES_LEFT_AT_CURRENT_PRICE == -1 %}
+                {% set badges_left_text = 'Limitless' %}
+            {% elif c.BADGES_LEFT_AT_CURRENT_PRICE <= 100 %}
+                {% set badges_left_text = 'Critical' %}
+            {% elif c.BADGES_LEFT_AT_CURRENT_PRICE <= 250 %}
+                {% set badges_left_text = 'Low' %}
+            {% else %}
+                {% set badges_left_text = 'Medium' if c.BADGES_LEFT_AT_CURRENT_PRICE <= 500 else 'High' %}
+            {% endif %}
+
+            if ($('.prereg-price-notice').size()) {
+                $('#reg-types').append("<div class='help-block col-sm-9 col-sm-offset-3'>Availability of tickets at this price: <strong>{{ badges_left_text }}</strong>. {{ macros.popup_link("http://magfest.org/faq#-KOpcFnNhdsUIqvvf6VB", "What does this mean?")|e }}</div>");
```

Comment:
I feel like "this price" should be "these prices", because we're going to have three badge prices listed: Attendee, Group, and Child.